documentation/rosetta-modelling-component.md

```diff
@@ -1041,20 +1041,20 @@ The Rosetta DSL supports [comparison operators](#comparison-operator) to functio
 
 If the `contains` operator is passed an expression that has single cardinality, that expression is treated as a list containing the single element or an empty list if the element is null.
 
-For the comparison operators, if either the left or right expression has multiple cardinality then the other side should have multiple cardinality. Otherwise if one side's expression has single cardinality, `all` or `any` should be used to qualify the list on the other side. At present only `any` is supported for `<` and `>` and only `all` for the other comparison operators.
+For the comparison operators, if either the left or right expression has multiple cardinality then the other side should have multiple cardinality. Otherwise if one side's expression has single cardinality, `all` or `any` should be used to qualify the list on the other side.
 
 The semantics for list comparisons are as follows:
 
 - `=`
-  - if both sides are lists, returns true if all items are equal
+  - if both sides are lists, returns true if the lists have the equal length and all items are equal when compared pairwise in the order of the lists
```

Comment:
"the equal length" --> "equal length" or "the same length" (I prefer the latter)